src/rust/src/wallet.rs

```diff
@@ -449,6 +459,7 @@ impl Wallet {
         // Generate the nullifier for the received note and add it to the nullifiers map so
         // that we can detect when the note is later spent.
         if let Some(fvk) = self.key_store.viewing_keys.get(&ivk) {
+            tracing::trace!("Adding decrypted note to the wallet");
```

Comment:
Consider logging more about the note and/or txid here.

Reply:
Alternatively, please document (in `help setlogfilter` and in https://zcash.readthedocs.io/en/latest/rtd_pages/security_warnings.html) that enabling trace-level events is always unsafe.

Reply:
`add_decrypted_note` is now instrumented.